packages/handlers/soap/src/index.ts

```diff
@@ -1,9 +1,20 @@
 import { MeshHandlerLibrary, YamlConfig } from '@graphql-mesh/types';
-import { soapGraphqlSchema } from 'soap-graphql';
+import { soapGraphqlSchema, createSoapClient } from 'soap-graphql';
+import { WSSecurityCert } from 'soap';
 
 const handler: MeshHandlerLibrary<YamlConfig.SoapHandler> = {
   async getMeshSource({ config }) {
-    const schema = await soapGraphqlSchema(config.wsdl);
+    const soapClient = await createSoapClient(config.wsdl)
+    const schema = await soapGraphqlSchema({
+      soapClient,
+    });
+    if (config.securityCert) {
```

Comment:
Maybe this should also support loading it from a string or interpolation? 

it will allow to use it without the need for `fs` 

So config will be something like:

```
securityCert: 
  publicKey: ${PUBLIC_KEY}
```

or

```
securityCert: 
  publicKey: {context.publicKey}
```